resources/dnf_module.rb

```diff
@@ -0,0 +1,104 @@
+resource_name :dnf_module
+provides :dnf_module
+
+unified_mode true
+
+property :module_name, String,
+          name_property: true,
+          description: 'Name of the module to install'
+
+property :options, [String, Array],
+          coerce: proc { |x| Array(x) },
+          default: [],
+          description: 'Any additional options to pass to DNF'
+
+action_class do
+  def supported?
+    (platform_family?('rhel') && node['platform_version'] >= 8) || platform?('fedora')
```

Comment:
Shouldn't `node['platform_version'] >= 8` be `node['platform_version'].to_i >= 8` ?

Reply:
On 15.3+ you are going to need to .to_i it. In the latest versions you don't need to anymore.

Reply:
Whoops, how did I miss that in my original pr.
Fix in #197.

Reply:
Thank you